src/App.js

```diff
@@ -130,6 +131,11 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
                 component={ReportsPartialsCampaigns}
               />
               <PrivateRoute path={['/billing/invoices']} exact component={InvoicesList} />
+              <PrivateRoute
+                path={'/ControlPanel/CampaignsPreferences/ContactPolicy'}
```

Comment:
Talking with @andresmoschini having two urls going to the same page and also having urls with uppercase is not so good idea. In some servers urls area case sensitive, and in some are not.  Also in most cases urls are without capitals, like in google. 
Also the meaning of this path, right now we copied from the old, but what happens if this section is moved outside of control panel?
Perhaps temporarily something along this line would be best, what do you think?
```suggestion
                path={'/sending-preferences/contact-policy'}
```
I will talk this with @jmonacchi too.

Reply:
Sounds good to me, let's remove this code too:
```
<RedirectWithQuery
  exact
  from="/PaneldeControl/PreferenciasdeCampana/PoliticadeContacto"
  to="/ControlPanel/CampaignsPreferences/ContactPolicy"
/>
```